Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -10994,18 +11035,17 @@ - (void)_completeImageAnalysisRequestForContextMenu:(CGImageRef)image requestIde
             return;
 
 #if USE(QUICK_LOOK)
-        strongSelf->_hasVisualSearchResults = hasVisualSearchResults;
-#endif
-#if USE(UICONTEXTMENU) && ENABLE(IMAGE_ANALYSIS_FOR_MACHINE_READABLE_CODES)
-        [strongSelf _updateContextMenuForMachineReadableCodeForImageAnalysis:result];
+        data->machineReadableCodeMenu = result.mrcMenu;
```

Comment:
I think we may still want to guard this behind `ENABLE(IMAGE_ANALYSIS_FOR_MACHINE_READABLE_CODES)`

Reply:
I seems cleaner to me without the compile-time guards, since `mrcMenu` is just guarded by `TARGET_OS_IOS` anyways. Is there a reason why we need this?

Reply:
Fixed!